articles/communication-services/concepts/call-automation/call-automation.md

```diff
@@ -72,29 +71,29 @@ Azure Communication Services uses Event Grid to deliver the [IncomingCall event]
 
 ### Pre-call actions
 
-These actions are performed before the destination endpoint listed in the IncomingCall event notification is connected. Web hook callback events only communicate the “answer” pre-call action, not for reject or redirect actions.  
+These actions are performed before the destination endpoint listed in the IncomingCall event notification is connected. Webhook callback events only communicate the “answer” pre-call action and not for reject or redirect actions.  
 
 **Answer**
-Using the IncomingCall event from Event Grid and Call Automation SDK, a call can be answered by your application. This action allows for IVR scenarios where an inbound PSTN call can be answered programmatically by your application. Other scenarios include answering a call on behalf of a user.
+Using the IncomingCall event from Event Grid and Call Automation SDK, yourt application can answer a call programmatically. This action enables IVR scenarios where an inbound PSTN call can be answered by your application. Other scenarios include answering a call on behalf of a user.
 
 **Reject**
-To reject a call means your application can receive the IncomingCall event and prevent the call from being connected to the destination endpoint.
+To reject a call, your application can receive the IncomingCall event and prevent the call from being connected to the destination endpoint.
```

Comment:
this is changing the sentence meaning - 
The sentence explains what rejecting a call means - can you reword this please ?